lib/DescriptionFileUtils.js

```diff
@@ -56,7 +80,7 @@ function loadDescriptionFile(
 						}
 						let json;
 						try {
-							json = JSON.parse(content);
+							json = JSON.parse(/** @type {string} */ (content));
```

Comment:
It could be a Buffer too, so the cast is not correct. Call .toString() that should work for Buffer and string